### ISSUE TITLE
Allow players to report no-shows.

### ISSUE THREAD

**Issue by silasary** [PennyDreadfulMTG/gatherling_fork#27]
This should probably just be a flag in the Match Listings, and not a fully automated thing.

But it's better than reporting 2-0 with no additional details.

**Comment by stash86** [comment]
You would need to add a new column orn matches table. I have done minor changes on the interface to add 5th result (opp not shows up), but I would need a new column to store the extra info

**Comment by bakert** [comment]
You can potentially do it without any db changes by having it perform a
drop + result entry. Bit dangerous to put that power in the hands of
players but I like the idea in theory.

**Comment by stash86** [comment]
Yup, that's the only way we can do it using current DB structure, but it;s a very dangerous route to take,

**Comment by stash86** [comment]
How about this? Let's say Player A vs Player B, and Player B doesn't show up. To use the current table structure, my proposal is 
- Save the result as "WO" instead of "A"
- Then later detect this "WO" result in host's match listing tab and gives it a flag
- Later when TO manually input the result, that "WO" will be replaced with "A" correctly

**Comment by stash86** [comment]
Is this enough? I can add an icon to make it more obvious I guess
![image](https://user-images.githubusercontent.com/13656154/44312231-dc39e500-a42f-11e8-9ada-def1fd8622f2.png)

**Comment by bakert** [comment]
What's "WO"?

**Comment by silasary** [comment]
I was wondering the same thing.

**Comment by stash86** [comment]
https://en.m.wikipedia.org/wiki/Walkover

Maybe I should use W.O. instead

On 20 Aug 2018, at 20.59, Thomas David Baker <notifications@github.com> wrote:

What's "WO"?

**Comment by bakert** [comment]
We have a tendency to use "no show"/"noshow" as the terminology. It's no
better than walkover but might stick with it for consistency.

**Comment by bakert** [comment]
Either way I'd need the whole word to understand.

**Comment by stash86** [comment]
I'm thinking of using an icon to show no show, but haven't found the perfect one

On 20 Aug 2018, at 21.27, Thomas David Baker <notifications@github.com> wrote:

Either way I'd need the whole word to understand.

**Comment by bakert** [comment]
"noshow" is only 6 chars. bit long compared to current UI but might be ok?

**Comment by stash86** [comment]
How about this?

![image](https://user-images.githubusercontent.com/13656154/44384431-1fdb3e80-a557-11e8-948a-e51bd75d5f25.png)

**Comment by bakert** [comment]
If this is just for hosts then we can expect them to learn what things mean
but I don't think it's completely obvious

I'd go with "Opponent no show" for any mouseover text

**Comment by stash86** [comment]
"W.O. 2-0" with "Opponent don't show up" tooltip in the host page and the deck page is possible, although to make this win recorded permanently instead of the simple 2-0 won would require careful edit of the related codes

**Comment by bakert** [comment]
Yeah it may not be worth doing in a hacky way but I leave that to you.
